### PR TITLE
Test clusters: convert x-pack qa tests

### PR DESCRIPTION
**Pull request by alpar-t**
This PR converts everything in x-pack qa that is not a bwc test and does not involve TLS.

### CODE REVIEW

File: x-pack/qa/multi-node/build.gradle [hunk 1]
```diff
@@ -1,30 +1,22 @@
...
   testCompile project(':x-pack:qa')
 }
 
-integTestCluster {
-  numNodes = 2
-  clusterName = 'multi-node'
+testClusters.integTest {
+  distribution = 'DEFAULT'
+  numberOfNodes = 2
+  //clusterName = 'multi-node'
```
Comment (mark-vieira) [inline x-pack/qa/multi-node/build.gradle]: Does this need to remain commented out? Can we remove this?

File: x-pack/qa/saml-idp-tests/build.gradle [hunk 2]
```diff
@@ -35,56 +37,44 @@ task setupPorts {
...
+    setting 'xpack.security.authc.realms.saml.shibboleth_native.authorization_realms', 'native'
+    setting 'xpack.security.authc.realms.native.native.order', '3'
 
-  extraConfigFile 'idp-metadata.xml', file(outputDir + "/idp-metadata.xml")
+    setting 'xpack.ml.enabled', 'false'
+    setting 'logger.org.elasticsearch.xpack.security', 'TRACE'
 
-  setupCommand 'setupTestAdmin',
-            'bin/elasticsearch-users', 'useradd', "test_admin", '-p', 'x-pack-test-password', '-r', "superuser"
+    extraConfigFile 'idp-metadata.xml', file(outputDir + "/idp-metadata.xml")
```
Comment (mark-vieira) [inline x-pack/qa/saml-idp-tests/build.gradle]: We can use a GString here: `file("${outputDir}/idp-metadata.xml")`.

File: x-pack/qa/security-example-spi-extension/build.gradle [hunk 3]
```diff
@@ -13,12 +14,16 @@ dependencies {
 }
 
 
-integTestRunner {
-    systemProperty 'tests.security.manager', 'false'
```
Comment (mark-vieira) [inline x-pack/qa/security-example-spi-extension/build.gradle]: Is this no longer needed?

**Review by mark-vieira — APPROVED** [review]
Minor comments, otherwise LGTM 👍

File: x-pack/qa/security-example-spi-extension/build.gradle (hunk 3 above)
Comment (atorok) [reply x-pack/qa/security-example-spi-extension/build.gradle]: yes, tests pass without it so we probably don't want to have it in an example.

**Pull request merged**